app/assets/v2/js/cart-ethereum-polygon.js

```diff
@@ -381,6 +382,14 @@ Vue.component('grantsCartEthereumPolygon', {
 
       let gasLimit = 0;
 
+      // fetch gas prices from polygon gas tracker
+      if (!this.polygon.gasPrices) {
+        const priceUrl = 'https://api.polygonscan.com/api?module=gastracker&action=gasoracle&apikey=I28K1DVQAWAISBSI146I71YQDBK6N1C9GJ';
```

Comment:
We might want to switch this out for a paid account or move this to the backend and set up a cron to store and update the `safeGasPrice` periodically.

This is going to get hammered if we call it every time a user hits the checkout.

Reply:
I'd say set a high default price and wrap this in a try catch for now

Reply:
Shouldn't we load the api key from the settings?

Reply:
Oh that is a nice catch ! @chibie i agree with nutrina here

Reply:
Yeah we should really be protecting that key - I would recommend moving this to the backend and returning the `safeGasPrice` as part of the context.

Reply:
@gdixon pushed a commit https://github.com/gitcoinco/web/pull/9733/commits/c2cfb70b9e1bb1f0a8bd3a58ff70de63c1f30aab

we should run `./manage.py fetch_gas_prices` after deploy